### ISSUE TITLE
ciao cool stuff;-) you can help me?!

### ISSUE THREAD

**Issue by saibai** [Alzy/alzyOSC#1]
I'm at the beginning, as I can make communicate another matrix "C", I tried changing both in LIVEOSC "alzyOSC" and in "alzyOSC-custom-module", but I think I'm wrong something ......

Grazie

**Comment by Alzy** [comment]
Hi Diego! I'm amazed you found my project lol. Very cool. Have you gotten the session working? What are you trying to accomplish? A third matrix?

**Comment by saibai** [comment]
Yes, it works very well ;-) I'm trying to make a third matrix work to create 4 instruments  ;-) so I can have more controls, I created 3 more pages where I inserted a keyboard and other various creative possibilities, but they are at the beginning  with python and js, I tried to insert the matrix "C" but in the end I don't work or always return the matrix "B".  I modified both the open stage control js, and the python you edit on Ableton Live

**Comment by Alzy** [comment]
Making a third matrix will be quite the project. I've since considered rewriting the project to be more flexible and performant. But if you want to give it a go you'll have to update the alzyOSC python script to clone one of the matrix code for the C track. You'll also have to initialize a third instance of the remote script in your Ableton settings. Remember to delete the .pyc files and reopen Ableton between saves as Ableton compiles the remote scripts on startup. Finally you'll have to edit the custom module js to handle the third matrix.

**Comment by saibai** [comment]
I had done everything, maybe you can help me better with the project I modified, and we can build one together ;-)

[AbletonLiveOSC_Project.zip](https://github.com/Alzy/alzyOSC/files/3861104/AbletonLiveOSC_Project.zip)

**Comment by Alzy** [comment]
Wow. Very cool! I'm very interested in continuing this venture with you and expanding on the project further! Happy to see someone shares the vision lol. Unfortunately I won't be able to help until at least after this weekend. I'm going to give this project its first official test run at a local show I am performing this Saturday. Very excited.

I took a quick glance at your code real quick though and here's a couple quick notes:

- In alzyOSC.py you should probably make a third if case in the init method to account for the third session.
- In alzyOSCSessionComponent your session C callbacks are not indented properly. They lie outside the scope of the class. Thus they will not work as expected. In python whitespace is very important.

**Comment by saibai** [comment]
thanks ;-) now I see what I can do, meanwhile I build the UI and I hope to continue this project together .... good adventure for Saturday.
we update ourselves when you have time ... nice to meet you

**Comment by Alzy** [comment]
Hey @saibai how's everything? Sorry for the late response lol
